_component_design/buttons.md

```diff
@@ -14,203 +14,193 @@ last-modified: 2017-08-17
 {% endfigure %}
 {% endcolumn %}
-</div>
 
-# Secondary Buttons
+</div>
 
+## Simple buttons
+{: .hxSectionTitle}
 <div class="hxRow">
-{% column left:"hxCol" %}
--   The secondary button can be used in conjunction with the primary button or
-    when there are multiple buttons on the same screen.
 
--   The same sizing rules apply to both secondary and primary buttons.
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+When using a simple button, specify both weight and size.
+
+- Weights:   
+    - Primary
+    - Secondary
+    - Teritiary
+- Sizes:
+    - Large
+    - Standard
+    - Compact
 
 {% endcolumn %}
-{% column right:"hxCol" %}
-{% figure [caption:"Secondary button sizes"] %}
-![]({{site.cdn_url}}/img/components/Buttons/buttons-secondary-sizes.svg){:width="100%"}
-{% endfigure %}
-{% endcolumn %}
-{% column right:"hxCol" %}
-{% figure [caption:"Secondary button states"] %}
-![]({{site.cdn_url}}/img/components/Buttons/buttons-secondary-actions.svg){:width="100%"}
+
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+{% figure [caption:"The three button weights"] [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/components/inputs-and-controls/buttons/buttons-simple.svg){:width="100%"}
 {% endfigure %}
 {% endcolumn %}
 </div>
 
-# Tertiary Buttons
-
+### Spacing and specs
+{: .hxSubSectionTitle}
 <div class="hxRow">
-{% column left:"hxCol" %}
--   Tertiary buttons can be used in conjunction with the primary button or with
-    a primary and secondary button.
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+For most desktop use cases, you should use a standard button size.   
 
--   Tertiary buttons should be used to deemphasize the action in comparison
-    with the actions of primary or secondary buttons.
+Large buttons should be used in mobile designs to accommodate the ideal touch target size. 
 
--   For example, use a tertiary button for "cancel" and a primary button to
-    emphasize "submit".
+Compact buttons should be used when you have severe space constraints, such as inside of a table or in a dashboard widget.
 
 {% endcolumn %}
-{% column right:"hxCol" %}
-{% figure [caption:"Tertiary button sizes"] %}
-![]({{site.cdn_url}}/img/components/Buttons/buttons-tertiary-sizes.svg){:width="100%"}
-{% endfigure %}
-{% endcolumn %}
-{% column right:"hxCol" %}
-{% figure [caption:"Tertiary button states"] %}
-![]({{site.cdn_url}}/img/components/Buttons/buttons-tertiary-actions.svg){:width="100%"}
+
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+{% figure [caption:"Button spacing specs"] [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/components/inputs-and-controls/buttons/buttons-specs.svg){:width="100%"}
 {% endfigure %}
 {% endcolumn %}
 </div>
 
-# Dropdowns
-
-Dropdowns inject additional actions into a primary, secondary, or tertiary
-button.
-
+### Primary buttons - simple
```

Comment:
for now, let's remove the "- simple" on these.  I feel like thats a distinction most designers would not stumble over, and making the distinction apparent may actually cause more pause about what it means.

Reply:
removed.